### ISSUE TITLE
Question: What's the main difference to Invidious ?

### ISSUE THREAD

**Issue by petrarca-arezzo** [TeamPiped/Piped#831]
### Official Instance

- [X] The bug is reproducable on the [official hosted instance](http://piped.kavin.rocks/) or is API related.

### Describe the bug

No bug - but couldn't really find any information what distinguishes Piped from Invidious. I think it would be nice to explain it (e.g. within the README.md or on the wiki pages).

### To Reproduce

Just search around - at least I couldn't find information about the difference(s) between Piped and Invidious.

### Expected behavior

Get some more information about the aim of Piped and the differences compared to Invidious.

### Logs/Errors

None

**Comment by FireMasterK** [comment]
https://piped-docs.kavin.rocks/docs/why/#invidious

Piped supports VP9 / Webm, the official VueJS frontend is a single page application, Piped takes a completely different approach/architecture, which is more optimized for CDNs, and uses a microservice architecture for certain components.

I'll add a wiki page whenever I can.

**Comment by Yolakalemowa** [comment]
> 

as someone who has no idea what any of that means lol... could someone possibly explain in layman terms?

**Comment by Yolakalemowa** [comment]
> https://piped-docs.kavin.rocks/docs/why/#invidious
> 
> Piped supports VP9 / Webm, the official VueJS frontend is a single page application, Piped takes a completely different approach/architecture, which is more optimized for CDNs, and uses a microservice architecture for certain components.
> 
> I'll add a wiki page whenever I can.

as someone who has no idea what any of that means lol... could someone possibly explain in layman terms?

**Comment by IronMaltese** [comment]
> as someone who has no idea what any of that means lol... could someone possibly explain in layman terms?
I made a small table comparing YouTube frontends if you want to see the main differences at https://github.com/IronMaltese/YouTube-Frontend-Comparison

**Comment by Yolakalemowa** [comment]
@IronMaltese 

that's pretty cool actually thanks
